src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy-details/rgw-multisite-sync-policy-details.component.ts

```diff
@@ -225,4 +280,67 @@ export class RgwMultisiteSyncPolicyDetailsComponent implements OnChanges {
       }
     });
   }
+
+  async openPipeModal(edit = false) {
+    const action = edit ? 'edit' : 'create';
+    const initialState = {
+      groupExpandedRow: this.expandedRow,
+      pipeSelectedRow: this.pipeSelection.first(),
+      action: action
+    };
+
+    this.modalRef = this.modalService.show(RgwMultisiteSyncPipeModalComponent, initialState, {
+      size: 'lg'
+    });
+
+    try {
+      const res = await this.modalRef.result;
+      if (res === 'success') {
+        this.loadData();
+      }
+    } catch (err) {}
+  }
```

Comment:
I don't really get what is this for

Reply:
once the modal is closed then I have to reload my datatable, its for that

Reply:
makes sense, thanks